crates/sovereign-sdk/module-system/sov-modules-rollup-blueprint/src/lib.rs

```diff
@@ -38,14 +38,14 @@ pub trait RollupBlueprint: Sized + Send + Sync {
     /// Context for Zero Knowledge environment.
     type ZkContext: Context;
     /// Context for Native environment.
-    type NativeContext: Context;
+    type NativeContext: Context + Send;
 
     /// Manager for the native storage lifecycle.
     type StorageManager: HierarchicalStorageManager<
         Self::DaSpec,
         NativeStorage = <Self::NativeContext as Spec>::Storage,
         NativeChangeSet = <Self::NativeContext as Spec>::Storage,
-    >;
+    > + Send;
```

Comment:
i don't know what thse + Sends would imply. 


@kpp @rakanalh is this ok?

Reply:
TBH, I did this change completely unaware of what the hell is going on. It didn't allow me to call `seq.resubmit_pending_commitments().await` in `create_new_sequencer` method until I did this. I believe it has smh. to due with this method mutating the `seq` asynchronously at the point the instance is initialized, but no idea in the end :).